apps/bidsificator/build.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+APP_NAME="bidsificator"
+APP_VERSION="1.0.0"
```

Comment:
is it used?

Reply:
you don't have to package it, only to tag it.

Reply:
That i can do